pkg/stanza/operator/helper/encoding.go

```diff
@@ -14,48 +14,44 @@ import (
 	"golang.org/x/text/transform"
 )
 
-// NewBasicConfig creates a new Encoding config
+// NewEncodingConfig creates a new Encoding config
 func NewEncodingConfig() EncodingConfig {
 	return EncodingConfig{
 		Encoding: "utf-8",
 	}
 }
 
-// EncodingConfig is the configuration of a Encoding helper
+// EncodingConfig is the configuration of an Encoding helper
 type EncodingConfig struct {
```

Comment:
Any reason not to also deprecate this struct (and `NewEncodingConfig`)? I may be missing something but looks like we could move the `Encoding` field directly into the `SplitterConfig`.

Reply:
Ok, I forgot `EncodingConfig` is used elsewhere as well. Let's just leave it then.